### PR TITLE
Update gsl/develop from develop 2021/10/25

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

- Remove interstitial variables for land and ice emissivity and update the land and ice emissivity in the routine setemis in CCPP
- Enable debugging features in auto-generated CCPP physics caps
    - Cleanup variables for Ferrier-Aligo and RRTMGP in GFS_typedefs.{F90,meta}
    - Contains bug fixes for multigases array dimensions in CCPP_typedefs.{F90,meta}
    - Contains additional bug fixes in CCPP metadata in GFS_typedefs.{F90,meta}
- Remove old, outdated and no longer functional regression tests `fv3_gsd_sar` and `fv3_gsd_sar_debug` (according to the RRFS developers, these tests are no longer required)

## Dependencies

https://github.com/NOAA-GSL/ccpp-physics/pull/111
https://github.com/NOAA-GSL/fv3atm/pull/113
https://github.com/NOAA-GSL/ufs-weather-model/pull/107

## Testing

Regression testing on Hera with Intel and GNU:

1. Using `rt_ccpp_dev.conf`, create new temporary baselines and verify against them. All tests pass.

[rt_ccpp_dev_hera_gnu_create.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420990/rt_ccpp_dev_hera_gnu_create.log)
[rt_ccpp_dev_hera_gnu_verify.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420991/rt_ccpp_dev_hera_gnu_verify.log)
[rt_ccpp_dev_hera_intel_create.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420992/rt_ccpp_dev_hera_intel_create.log)
[rt_ccpp_dev_hera_intel_verify.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420993/rt_ccpp_dev_hera_intel_verify.log)

2. Testing against existing/official baselines using `rt.conf`/`rt_gnu.conf`: all tests pass, except those that are expected to fail due to b4b mismatches (because of changes in gsl/develop that are not yet in the authoritative repositories). All of these tests run to completion.

[rt_hera_gnu_verify_against_existing.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420996/rt_hera_gnu_verify_against_existing.log)
[rt_hera_gnu_verify_against_existing_fail_test.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420998/rt_hera_gnu_verify_against_existing_fail_test.log)
[rt_hera_intel_verify_against_existing.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7420999/rt_hera_intel_verify_against_existing.log)
[rt_hera_intel_verify_against_existing_fail_test.log](https://github.com/NOAA-GSL/ufs-weather-model/files/7421000/rt_hera_intel_verify_against_existing_fail_test.log)

These PRs are ready to merge.

### CODE REVIEW

**Review by hannahcbarnes — APPROVED** [review]
From what I can tell these changes are acceptable.

File: .gitmodules [hunk 1]
```diff
@@ -1,15 +1,17 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-GSL/fv3atm
-  branch = gsl/develop
+  #url = https://github.com/NOAA-GSL/fv3atm
+  #branch = gsl/develop
+  url = https://github.com/climbfuji/fv3atm
```
Comment (joeolson42) [inline .gitmodules]: Are you sure you want this change in gsl/develop?

**Review by joeolson42 — APPROVED** [review]
Just the one comment/question. All else passes my tired eyes.

File: .gitmodules (hunk 1 above)
Comment (climbfuji) [reply .gitmodules]: This is only temporary, as usual for every commit, and will be reverted once fv3atm is merged.

**Pull request merged**